src/main/java/com/logistic/project/model/Order.java

```diff
@@ -1,4 +1,31 @@
 package com.logistic.project.model;
 
+import jakarta.validation.constraints.Min;
+import lombok.Data;
+
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+@Data
 public class Order {
+    private Set<Warehouse> warehouses;
```

Comment:
`=new HashSet<>()`

Reply:
Idea shows warning that it's redundant code